eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/batch/processors/VariantToVariantContextProcessor.java

```diff
@@ -96,6 +97,7 @@ private String getSequenceName(String contig) {
                 case VARIANT_CLASS_KEY:
```

Comment:
No part of the new code but I think it worth correcting this. When the attribute key is VARIANT_CLASS_KEY it will add the MAPPING_WEIGHT_KEY as we don't have a break for the empty cases

Reply:
as discussed offline, the code is actually fine